RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -284,6 +284,25 @@
 from Configuration.Eras.Modifier_phase2_timing_layer_cff import phase2_timing_layer
 phase2_timing_layer.toModify(highPtTripletStepTracks, TrajectoryInEvent = True)
 
+highPtTripletStepLSTpTracks = highPtTripletStepTracks.clone(
+     src           = 'highPtTripletStepTrackCandidates:pTCsLST'
+)
+highPtTripletStepLSTT5Tracks = highPtTripletStepTracks.clone(
+     src           = 'highPtTripletStepTrackCandidates:t5TCsLST'
+)
+_highPtTripletStepTracks_LST = RecoTracker.FinalTrackSelectors.trackListMerger_cfi.trackListMerger.clone(
+    TrackProducers     = ['highPtTripletStepLSTpTracks',
+                          'highPtTripletStepLSTT5Tracks'],
+    hasSelector        = [1,0],
+    indivShareFrac = cms.vdouble(0.1,0.1),
+    selectedTrackQuals = ['highPtTripletStepSelector:highPtTripletStep',
+                          'highPtTripletStepSelectorLSTT5:highPtTripletStepLSTT5'],
+    copyExtras = cms.untracked.bool(True),
+    copyMVA = cms.bool(False),
+    setsToMerge        = [cms.PSet( tLists=cms.vint32(0,1), pQual=cms.bool(True) )]
+)
+(trackingPhase2PU140 & trackingLST).toReplaceWith(highPtTripletStepTracks, _highPtTripletStepTracks_LST)
```

Comment:
`highPtTripletStepTracks` are used in
https://github.com/SegmentLinking/cmssw/blob/f9e0bcf7dc1af26f6d86843f43a023d303379317/RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py#L36-L41

To make things work in this setup, I had to create the `highPtTripletStepTracks` from the union of `highPtTripletStepLSTpTracks` and `highPtTripletStepLSTT5Tracks` but with a specific ID in each case, to emulate the tracks that are produced at the end. Then, in
https://github.com/SegmentLinking/cmssw/blob/ccace1ec3a5e08d55deeae2a2b8483c5ed46be79/RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py#L42-L47
I still used `highPtTripletStepTracks` but without overriding the track quality, since this is already applied at the stage where I join the two track collections.
Is this a reasonable way to do it, or should the `highPtTripletStepTracks` not have a track quality applied to it to begin with?